create/content_template_handler.go

```diff
@@ -97,8 +97,7 @@ func executeArcheTypeAsTemplate(s *hugolib.Site, name, kind, targetPath, archety
 
 	if name == "" {
 		name = f.TranslationBaseName()
-
-		if name == "index" || name == "_index" {
+		if f.FileInfo().Meta().Classifier().IsBundle() || name == "index" {
```

Comment:
There are 3 test failures if I remove the "index" test here: `TestNewContent/lang-11` through `TestNewContent/lang-13`.